code/game/objects/structures/xeno.dm

```diff
@@ -169,6 +191,35 @@
 	visible_message("<span class='warning'>[hugger] gets out of [src]!</span>")
 	hugger = null
 
+/obj/effect/alien/resin/trap/proc/acid_activate()
+	if(gastrap)
```

Comment:
Dont really need this if, as its handled via the switch

Reply:
alrighty, Ill remove the if(gastrap)